# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1357241358 18000
#      Thu Jan 03 14:29:18 2013 -0500
# Branch stable
# Node ID 56b74fffac32c9083b75d596a949ea5809e0559a
# Parent  7f4082b00a99edb0780292857635cc4d70407be1
maint: update gnulib subrepo

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,1 @@
-0505376775c93711e62a4fa5cc0112501b0b737e gnulib-hg
+7ebb2c82d76fcf8b572ace5a9037d828757a2d6d gnulib-hg
